src/api/route-services/plots-tables.js

```diff
@@ -28,6 +28,28 @@ class PlotsTablesService {
     return tableData;
   }
 
+  async updatePlotData(experimentId, plotUuid, plotData) {
+    const marshalledData = convertToDynamoDbRecord({ ':d': plotData });
+
+    logger.log('debugging marshalled data', marshalledData);
```

Comment:
can you change the log message to something more descriptive of what happened like `plotData is marshalled` or something similar?

Reply:
Sorry, this was a left over from debugging